hack/unwanted-dependencies.json

```diff
@@ -14,6 +14,7 @@
       "github.com/bketelsen/crypt": "unused, crypto",
       "github.com/containerd/cgroups": "standardize on single cgroups library from runc, refer #128157",
       "github.com/davecgh/go-spew": "refer to #103942",
+      "github.com/evanphx/json-patch/v5": "refer to #91622#issuecomment-637087847",
```

Comment:
full link to https://github.com/kubernetes/kubernetes/pull/91622#issuecomment-637087847 ?

and this block is only as long as it is incompatible... if v5 became compatible or allowed options to be compatible with v4, we'd probably update and use it

Reply:
updated and included in https://github.com/kubernetes/kubernetes/pull/132499

/close